hack/update-release-parameters.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+# Accept a file path for param yaml.
+releaseParams=$1
+
+# Get the paths to all files that can be automatically updated for a release.
+readarray updates < <(yq e -o=j -I=0 '.updates[]' $releaseParams)
```

Comment:
♻️ With `set -e`, it is good to break pipelines into separate commands. With the way this is arranged now, the script will continue if `yq` fails.

Below, the first line will bail when `yq` does. The `<<<` redirection passes a single word to command's stdin.

```suggestion
updates=$(yq e -o=j -I=0 '.updates[]' $releaseParams)
readarray updates <<< "$updates"
```